scripts/core/INTRO.rst

```diff
@@ -207,23 +207,23 @@ By default, there is a guarantee that *error* messages are flushed immediately.
 Loggers redirect messages to *stdout*, *stderr*, or a file (default: *stderr*).
 
 All of these logging options can be set with **UR_LOG_LOADER** and **UR_LOG_NULL** environment variables described in the **Environment Variables** section below.
-Both of these environment variables have the same syntax for setting logger options::
```

Comment:
I belive only removing the second `:` and leaving the rest as it was before will also fix the error while maintaining the original formatting.

Reply:
That's right, thanks.